.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -12,6 +12,21 @@ body:
       options:
         - label: I have searched the existing issues
           required: true
+  - type: dropdown
+    attributes:
+      label: Product
+      description: Which page or product does this relate to?
+      multiple: false
+      options:
+        - Farms
+        - Syrup Pools
+        - Lottery
+        - Predictions
```

```suggestion
        - Prediction
```

Throughout the codebase and translations, we roll without an 's'